qemu/tests/sgx_basic.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from virttest import error_context
+from provider.sgx import SGXTestCapability
+from virttest import env_process
+from virttest.utils_misc import verify_dmesg
+from provider.sgx import SGXChecker
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sgx basic test:
+    1. Check host sgx capability
+    2. Boot sgx VM
+    3. Check sgx qmp cmd
+    4. Verify sgx flags and epc size
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    error_context.context("Start sgx test", test.log.info)
+    timeout = float(params.get("login_timeout", 240))
+
+    sgx_cap = SGXTestCapability(test, params)
+    host_epc_size = sgx_cap.valid_host_sgx_cap()
+    params["host_epc_size"] = host_epc_size
+    if params.get("monitor_expect_nodes"):
+        sgx_cap.valid_host_numa_node()
+    params['start_vm'] = "yes"
+
+    vm_name = params["vms"]
+    env_process.preprocess_vm(test, params, env, vm_name)
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login(timeout=timeout)
+    verify_dmesg()
+    dmesg_output = session.cmd_output(params["guest_sgx_check"],
+                                      timeout=240).strip()
+    params['dmesg_output'] = dmesg_output
+    session.close()
+
+    test_check = SGXChecker(test, params, vm)
+    test_check.guest_dmsg_sgx_verify()
+    test_check.host_qmp_sgx_verify()
+    test_check.guest_qmp_sgx_verify()
+    error_context.context("Destroy guest.", logging.info)
```

Comment:
test.log.info, like line 23

Reply:
Thanks Xueqiang, done. change to test.log.info